src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_common.py

```diff
@@ -71,6 +76,104 @@ def firewall_rule_create_func(client, resource_group_name, server_name, firewall
         parameters)
 
 
+def migration_create_func(cmd, client, subscription_id, resource_group_name, server_name, body, migration_id=None):
+
+    if migration_id is None:
+        # Convert a UUID to a string of hex digits in standard form
+        migration_id = str(uuid.uuid4())
+
+    r = send_raw_request(cmd.cli_ctx, "put", "https://management.azure.com/subscriptions/{}/resourceGroups/{}/providers/Microsoft.DBforPostgreSQL/flexibleServers/{}/migrations/{}?api-version=2020-02-14-privatepreview".format(subscription_id, resource_group_name, server_name, migration_id), None, None, body)
```

Comment:
there is no SDK available for the operation, so we have to user raw request?

Reply:
Yes, that is correct.

Reply:
Will there be SDK support in the future?